archetypes/camel-archetype-spring-boot/src/main/resources/archetype-resources/src/main/resources/application.properties

```diff
@@ -35,7 +35,7 @@ timer.period = 2000
 management.endpoints.web.exposure.include=info,health,camelroutes
 
 # all access to actuator endpoints without security
-management.security.enabled = false
+#management.security.enabled = false #is no longer customizable with SB 2.0
```

Comment:
Should we maybe just delete all of this as its not possible with SB2 like this way?

Reply:
considering support of former versions i just kept them commented. but i guess by 2.22.x we will drop and not dually support SB 1.X. so you are right. i will remove them.